.github/workflows/stale_issue.yml

```diff
@@ -0,0 +1,47 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+# This is every day at 10am
+on:
+  schedule:
+    - cron: "0 10 * * *"
+
+jobs:
+  cleanup:
+    runs-on: ubuntu-latest
+    name: Stale issue job
+    steps:
+      - uses: aws-actions/stale-issue-cleanup@v3
+        with:
+          # Setting messages to an empty string will cause the automation to skip
+          # that category
+          ancient-issue-message: Greetings! It looks like this issue hasn’t been active in longer than three years. We encourage you to check if this is still an issue in the latest release. Because it has been longer than three years since the last update on this, and in the absence of more information, we will be closing this issue soon. If you find that this is still a problem, please feel free to provide a comment to prevent automatic closure, or if the issue is already closed, please feel free to reopen it.
+          stale-issue-message: Greetings! It looks like this issue hasn’t been active in longer than a week. We encourage you to check if this is still an issue in the latest release. Because it has been longer than a week since the last update on this, and in the absence of more information, we will be closing this issue soon. If you find that this is still a problem, please feel free to provide a comment or add an upvote to prevent automatic closure, or if the issue is already closed, please feel free to open a new one.
```

Comment:
What distinguishes ancient from stale? These messages are the same aside from the timeline?

Reply:
Yeah, ancient is for a longer time frame than stale and ancient also allows for exemption based on input minimum number of votes, see line 42. We can adjust as needed as we get more issues.